projects.md

```diff
@@ -844,6 +844,21 @@ Specific tasks:
 - Add support for visualizing metadata, such as annotations and labels.
 - Create UI elements for plotting statistical summaries computed in real-time.
 
+### Investigate the dynamic linking landscape
+
+- **Topics:** `Operating Systems` `Compilers` `Linux` `Package Management` `NixOS`
+- **Skills:** Experience with systems programming and Linux familiarity
+- **Difficulty:** Moderate to Challenging
+- **Mentors:** [Farid Zakaria](mailto:fmzakari@ucsc.edu)
```

Comment:
@trws if you have bandwidth and want to co-mentor let me know.
Thank you for help mentoring me thus far :)

Reply:
Sure, I'd be happy to.  Always happy to mentor when I get the opportunity. =)